### ISSUE TITLE
DDC kernel buildbot step on windows uses wrong path

### ISSUE THREAD

**Issue by karlklose** [dart-lang/sdk#31539]
Stack trace:
```   dartdevk arguments: --packages=E:\b\build\slave\ddc-win-release-be\build\sdk\out\ReleaseX64\dart-sdk\.packages --dart-sdk-summary E:\b\build\slave\ddc-win-release-be\build\sdk\out\ReleaseX64\gen\utils\dartdevc\ddc_sdk.dill -o E:/b/build/slave/ddc-win-release-be/build/sdk/out/ReleaseX64/generated_compilations/dartdevk-checked-strong-sdk/multitest_language_2_arg_param_trailing_comma_test_9/E:\b\build\slave\ddc-win-release-be\build\sdk\out\ReleaseX64\generated_tests\language_2\arg_param_trailing_comma_test_9.js E:\b\build\slave\ddc-win-release-be\build\sdk\out\ReleaseX64\generated_tests\language_2\arg_param_trailing_comma_test_9.dart -s E:\b\build\slave\ddc-win-release-be\build\sdk\out\ReleaseX64\gen\utils\dartdevc\pkg\async_helper.dill -s E:\b\build\slave\ddc-win-release-be\build\sdk\out\ReleaseX64\gen\utils\dartdevc\pkg\collection.dill -s E:\b\build\slave\ddc-win-release-be\build\sdk\out\ReleaseX64\gen\utils\dartdevc\pkg\expect.dill -s E:\b\build\slave\ddc-win-release-be\build\sdk\out\ReleaseX64\gen\utils\dartdevc\pkg\js.dill -s E:\b\build\slave\ddc-win-release-be\build\sdk\out\ReleaseX64\gen\utils\dartdevc\pkg\matcher.dill -s E:\b\build\slave\ddc-win-release-be\build\sdk\out\ReleaseX64\gen\utils\dartdevc\pkg\meta.dill -s E:\b\build\slave\ddc-win-release-be\build\sdk\out\ReleaseX64\gen\utils\dartdevc\pkg\path.dill -s E:\b\build\slave\ddc-win-release-be\build\sdk\out\ReleaseX64\gen\utils\dartdevc\pkg\stack_trace.dill -s E:\b\build\slave\ddc-win-release-be\build\sdk\out\ReleaseX64\gen\utils\dartdevc\pkg\unittest.dill
    dart --version: 2.0.0-edge.faa4ff94081af6c0b30f84845b513ead48bae6b1 (Mon Dec 4 22:57:04 2017 +0000) on "windows_x64"
Unhandled exception:
FileSystemException(uri=e:; message=PhysicalFileSystem only supports file:* URIs)
#0      PhysicalFileSystem.entityForUri (package:front_end/src/api_prototype/physical_file_system.dart:24)
#1      MultiRootFileSystemEntity._resolveEntity (package:front_end/src/multi_root_file_system.dart:67)
<asynchronous suspension>
#2      MultiRootFileSystemEntity.delegate (package:front_end/src/multi_root_file_system.dart:55)
<asynchronous suspension>
#3      MultiRootFileSystemEntity.exists (package:front_end/src/multi_root_file_system.dart:73)
<asynchronous suspension>
#4      ProcessedOptions.validateOptions (package:front_end/src/base/processed_options.dart:200)
<asynchronous suspension>
#5      generateKernelInternal (package:front_end/src/kernel_generator_impl.dart:45)
<asynchronous suspension>
#6      generateKernel.<anonymous closure> (package:front_end/src/kernel_generator_impl.dart:32)
<asynchronous suspension>
#7      CompilerContext.runInContext.<anonymous closure> (package:front_end/src/fasta/compiler_context.dart:89)
#8      _rootRun (dart:async/zone.dart:1126)
#9      _CustomZone.run (dart:async/zone.dart:1023)
#10     runZoned (dart:async/zone.dart:1501)
#11     CompilerContext.runInContext (package:front_end/src/fasta/compiler_context.dart:89)
#12     CompilerContext.runWithOptions (package:front_end/src/fasta/compiler_context.dart:96)
#13     generateKernel (package:front_end/src/kernel_generator_impl.dart:31)
<asynchronous suspension>
#14     compile (package:front_end/src/api_unstable/ddc.dart:104)
<asynchronous suspension>
#15     _compile (package:dev_compiler/src/kernel/command.dart:117)
<asynchronous suspension>
#16     compile (package:dev_compiler/src/kernel/command.dart:32)
<asynchronous suspension>
#17     runBatch (file:///E:/b/build/slave/ddc-win-release-be/build/sdk/pkg/dev_compiler/bin/dartdevk.dart:42)
<asynchronous suspension>
#18     main (file:///E:/b/build/slave/ddc-win-release-be/build/sdk/pkg/dev_compiler/bin/dartdevk.dart:17)
<asynchronous suspension>
#19     _startIsolate.<anonymous closure> (dart:isolate-patch/dart:isolate/isolate_patch.dart:275)
#20     _RawReceivePortImpl._handleMessage (dart:isolate-patch/dart:isolate/isolate_patch.dart:163)
--- To debug locally, run:
E:\b\build\slave\ddc-win-release-be\build\sdk\tools\sdks\win\dart-sdk\bin\dart.exe E:\b\build\slave\ddc-win-release-be\build\sdk\tools\testing\dart\http_server.dart -p 60907 -c 60908 --network 127.0.0.1 --build-directory=E:\b\build\slave\ddc-win-release-be\build\sdk\out\ReleaseX64\ --runtime=chrome --packages=E:\b\build\slave\ddc-win-release-be\build\sdk\.packages
--- Re-run this test:
python tools/test.py -m release -c dartdevk -r chrome --checked --strong --use-sdk language_2/arg_param_trailing_comma_test/9
Done dartdevk-chrome-checked release_x64 language_2/arg_param_trailing_comma_test/9: fail
```

**Comment by athomas** [comment]
Possibly related https://github.com/dart-lang/sdk/commit/514c0a99381df18d67e22619cf8cee3c7b928709

**Comment by jensjoha** [comment]
Actually I think it was fixed in https://github.com/dart-lang/sdk/commit/ab2271f4f8ab786c5e194169832a497f0fa712ab --- but we're going to find out soon-ish.

**Comment by athomas** [comment]
The builder has been green for a while a now.

**Comment by jensjoha** [comment]
Yes, this particular thing was fixed in 83057c42b11355160c4e962855f5afffe93f861f (i.e. not in the above mentioned one).
